internal/engine/ingester/artifact/artifact.go

```diff
@@ -36,13 +44,39 @@ const (
 // Ingest is the engine for a rule type that uses artifact data ingest
 // Implements enginer.ingester.Ingester
 type Ingest struct {
+	ghCli provifv1.GitHub
+
+	testOverrideVerifier *verifier.Verifier
+}
+
+type verification struct {
+	IsSigned          bool   `json:"is_signed"`
+	IsVerified        bool   `json:"is_verified"`
+	Repository        string `json:"repository"`
+	Branch            string `json:"branch"`
+	WorkflowName      string `json:"workflow_name"`
+	RunnerEnvironment string `json:"runner_environment"`
+	CertIssuer        string `json:"cert_issuer"`
```

Comment:
should we also add the raw `in-toto` data here? It would be handy in case we want to do something more custom in the future.

Reply:
We discussed this but were not sure how to handle if the format of the in-toto provenance statement had changed. Would it be a "use at your own risk" kind of a thing to enable custom rules? Then I'd be fine with it.

Reply:
I think a "use at your own risk" kind of thing would be fine